diff -up ptouch-driver-1.3/rastertoptch.c.omv~ ptouch-driver-1.3/rastertoptch.c
--- ptouch-driver-1.3/rastertoptch.c.omv~	2019-03-22 01:45:16.167306677 +0100
+++ ptouch-driver-1.3/rastertoptch.c	2019-03-22 01:45:24.205596139 +0100
@@ -682,7 +682,7 @@ emit_job_cmds (job_options_t* job_option
  * @param do_cut     Emit codes to actually cut
  * @param do_mirror  Emit codes to mirror print
  */
-inline void
+static void
 emit_feed_cut_mirror (bool do_feed, unsigned feed,
                       bool do_cut,
                       bool do_mirror) {
@@ -895,7 +895,7 @@ const unsigned char mirror [0x100] = {
  * @return                    0 if entire line is empty (zeros)
  *                            nonzero if line contains nonzero pixels
  */
-inline int
+static int
 generate_emit_line (unsigned char* in_buffer,
                     unsigned char* out_buffer,
                     int buflen,
@@ -971,7 +971,7 @@ generate_emit_line (unsigned char* in_bu
  * @param job_options   Job options
  * @param page_options  Page options
  */
-inline void
+static void
 flush_rle_buffer (job_options_t* job_options,
                   page_options_t* page_options) {
 #ifdef DEBUG
@@ -1057,7 +1057,7 @@ flush_rle_buffer (job_options_t* job_opt
  * @param page_options  Page options
  * @param bytes         Number of bytes required.
  */
-inline void
+static void
 ensure_rle_buf_space (job_options_t* job_options,
                       page_options_t* page_options,
                       unsigned bytes) {
@@ -1128,7 +1128,7 @@ ensure_rle_buf_space (job_options_t* job
  * can cause the RLE representation to be longer (by 1 byte) than the
  * corresponding input sequence in buf.
  */
-inline void
+static void
 RLE_store_line (job_options_t* job_options,
                 page_options_t* page_options,
                 const unsigned char* buf, unsigned buf_len) {
@@ -1242,7 +1242,7 @@ RLE_store_line (job_options_t* job_optio
  * @param empty_lines     Number of empty lines to store
  * @param xormask         The XOR mask for negative printing
  */
-inline void
+static void
 RLE_store_empty_lines (job_options_t* job_options,
                        page_options_t* page_options,
                        int empty_lines,
